### DISCUSSION TITLE
unhandled run time error

### DISCUSSION THREAD

**Discussion by Yesh-hacker** [community/community#145297] [npm]
1 of 1 error
Next.js (15.0.3)

Unhandled Runtime Error

Error: Failed to call `useTranslations` because the context from `NextIntlClientProvider` was not found.

This can happen because:
1) You intended to render this component as a Server Component, the render
   failed, and therefore React attempted to render the component on the client
   instead. If this is the case, check the console for server errors.
2) You intended to render this component on the client side, but no context was found.
   Learn more about this error here: https://next-intl-docs.vercel.app/docs/environments/server-client-components#missing-context

Source
app\[locale]\page.tsx (7:29) @ Home

   5 |
   6 | const Home = () => {
>  7 |   const t = useTranslations("Index");
     |                             ^
   8 |   return <h1>{t("title")}</h1>;
   9 | };
  10 |

**Comment by ELHart05** [comment]
can you share the full page code?
